microservices/cart/src/cli.py

```diff
@@ -0,0 +1,53 @@
+"""
+Copyright (C) 2021 Clariteia SL
+
+This file is part of minos framework.
+
+Minos framework can not be copied and/or distributed without the express permission of Clariteia SL.
+"""
+import logging
+from pathlib import (
+    Path,
+)
+from typing import (
+    Optional,
+)
+
+import typer
+from minos.common import (
+    EntrypointLauncher,
+    MinosConfig,
+)
+
+logging.getLogger("aiohttp.access").setLevel(logging.WARNING)
+
+app = typer.Typer()
+
+
+class _MyEntrypointLauncher(EntrypointLauncher):
+    async def _setup(self):
+        import src
+
+        await self.injector.wire(modules=[src] + self._internal_modules)
```

Comment:
You don't need to inherit from `EntrypointLauncher` anymore, on latest `minos-microservice-common` versions the `external_modules` is available to inject the `src` module: 
```python
EntrypointLauncher.from_config(file_path, external_modules=[sys.modules["src"]])
```

Reply:
Done!